packages/api/src/modules/file/resolvers.ts

```diff
@@ -118,5 +118,27 @@ async function updateFile(_parent, args: IUpdateFileArgs, context) {
   return file;
 };
 
-export const mutations = { createFile, updateFile };
+interface IDeleteFileArgs {
+  projectId: string;
+  fileId: string;
+}
+
+async function deleteFile(_parent, args: IDeleteFileArgs) {
+  const file = await File.findOne({ _id: args.fileId });
+
+  if (!file) {
```

Comment:
Vamos optar por estourar um erro quando o arquivo não for encontrado? Ou não fazer nada?

Reply:
Eu acho que pode deixar o erro, pq eu particularmente gosto de dar feedback pra quem ta usando...imagina uma pessoa chega no postman e faz uma request e ai n acontece nada. Ele n tem feedback de pq n aconteceu nada.